MANIFESTO.md

```diff
@@ -1 +1,6 @@
 
+##Community
+
+Mankind did not evolve by sitting alone in the dark. For the software industry to really grow in the Caribbean we need collaboration, knowledge and skill sharing. Avoid platforms and tools that do not respect communities that form around them or worse; shun them.
+
+Community does not only mean software developers, after all we need to solve somebody's problems. Developers should therefore take time to interact with and integrate with the rest of society.
```

Comment:
> Don't make assumptions though, there is no mention there of Caribbean Developers or accusations of anything.

Firstly, the previous paragraph was about the Caribbean software industry so if you don't state otherwise the reader is going to assume the second paragraph refers to us as well. Secondly, by recommending a course of action we are implicitly critiquing the status quo. When you say 

> Developers should therefore take time to interact with and integrate with the rest of society.

it can be interpreted as implying that developers don't already do this. If a reader is making assumptions that we did not foresee I would recommend that our writing be more direct.

> It's merely a point of view on what it takes to improve the quality of the industry locally/regionally

Manifestos are opinionated so I don't have any issues with a point of view being expressed. However, we can get into problems when the points of view are not concrete. The reader needs us to provide reasons for our points of view, because those who don't think the same as we do will need some convincing. My feedback was intended to make you aware of unanswered questions that were raised in my mind after reading.